packages/demo-wallet-shared/helpers/getNetworkConfig.ts

```diff
@@ -7,7 +7,6 @@ interface NetworkItemConfig {
 }
```

Comment:
Developers can manually update this file locally to use another Stellar network.